lib/mi_bridges/driver.rb

```diff
@@ -61,10 +66,15 @@ def run
       page_classes = SIGN_UP_FLOW + APPLY_FLOW
 
       page_classes.each do |klass|
-        page = klass.new(@snap_application)
-        page.setup
-        page.fill_in_required_fields
-        page.continue
+        begin
+          page = klass.new(@snap_application)
+          page.setup
+          page.fill_in_required_fields
+          page.continue
+        rescue StandardError => e
+          binding.pry if ENV["DEBUG_DRIVE"]
```

Comment:
nice! do we need this *and* DebuggerPage?

Reply:
I think so, this captures weird ass exceptions, whereas debugger page gives you space to fill in the page manually or via the context of a page.